### PR TITLE
Add query vars to screen option

### PR DESCRIPTION
**Pull request by seromenho**

### CODE REVIEW

File: js/interface.js [hunk 1]
```diff
@@ -63,10 +64,13 @@ $('form').submit(function (event) {
   if (data.url && !data.url.match(/^[A-z]+:/i)) {
     data.url = 'http://' + data.url;
   }
+console.log(data);
```
Comment (squallstar) [inline js/interface.js]: console.log

Comment (seromenho) [reply js/interface.js]: ups

**Review by tonytlwu — CHANGES_REQUESTED** [review]
@hcarneiro Please confirm (and tweak if necessary) the UI and flow before we merge this for deployment

**Comment by hcarneiro** [comment]
@tonytlwu - Revision done, see here: http://recordit.co/QjL2dRlIvc

**Review by tonytlwu — APPROVED** [review]

**Review by hcarneiro — APPROVED** [review]

**Pull request merged**